plugins/woocommerce-admin/client/products/fields/attribute-field/attribute-field.tsx

```diff
@@ -167,14 +176,20 @@ export const AttributeField: React.FC< AttributeFieldProps > = ( {
 				<Button
 					variant="secondary"
 					className="woocommerce-attribute-field__add-attribute"
-					onClick={ () => setShowAddAttributeModal( true ) }
+					onClick={ () => {
+						recordEvent( 'product_add_attribute_button' );
+						setShowAddAttributeModal( true );
+					} }
 				>
 					{ __( 'Add attribute', 'woocommerce' ) }
 				</Button>
 			</ListItem>
 			{ showAddAttributeModal && (
 				<AddAttributeModal
-					onCancel={ () => setShowAddAttributeModal( false ) }
+					onCancel={ () => {
+						recordEvent( 'product_modal_attribute_cancel_button' );
```

Comment:
Suggestion: `product_add_attributes_modal_cancel_button_click`

Also, since this same Tracks event name is used above, it should be stored in a `const` to ensure that the same event name is always used.

Reply:
Good idea! I added it in the commit [66d1d13](https://github.com/woocommerce/woocommerce/pull/35262/commits/66d1d131bc7ed26f6fd284b1ed030fd40b4a20fb)